### ISSUE TITLE
Change state of request from Unopened to Intake in Progress

### ISSUE THREAD

**Issue by mpilchar** [bcgov/foi-flow#386]
* As an Intake analyst
* I want a dropdown menu with the stages of a request cycle
* so that I can change the state of the request to Intake in Progress

**Assumptions & Scope**	
What are the assumptions for this story?
- Disable the "unopened" option from menu
- Validation happens
- Request stays "unopened" when I click on it - becomes "open" only once selected from dropdown menu
- Saving or using dropdown menu automatically changes state to "intake in progress"
-Follows states diagram, will only allow one state forward or back from current state: https://miro.com/app/board/o9J_l9AKkos=/

What is in and not in scope?
Change of dropdown will not save to db - to save to db, user needs to click a "save" button

**Acceptance Criteria**	

**Scenario 1: Unopened request Dropdown menu**
* GIVEN I am an intake analyst
* WHEN I I am on an unopened request
* THEN the state menu at the top of the 'Unopened' Request will say 'Unopened'

**Scenario 2: Click on Dropdown menu for 'Unopened' request**
* GIVEN an intake analyst is on an 'Unopened' request
* WHEN click the state dropdown menu
* THEN they will see these options: Intake in Progress

**Scenario 3: Change to Intake in Progress**
* GIVEN the intake analyst has clicked on the state dropdown menu for an 'Unopened' request
* WHEN ... I select "Intake in Progress" from the dropdown menu
* THEN the state of the request will be update to "Intake in Progress"
* AND the drop menu will close
* AND the menu will read "Intake in Progress"

...

**Dependencies? What is the impact of this dependency? (If so, link dependency in the ticket, make it visible in a team´s backlog)**

**Validation Rules? (If yes, list here)**	

**Design**
https://jacklyn808742.invisionapp.com/console/share/3W2O64AP5C/663872990

**Definition of Ready**

1. [ ] Is there a well articulated User Story?
2. [ ] Is there Acceptance Criteria that covers all scenarios (happy/sad paths)?
3. [ ] If there is a user interface, is there a design? 
4. [ ] Does the user story need user research/validation?
5. [ ] Does this User Story needs stakeholder approval?
6. [ ] Design / Solution accepted by Product Owner
7. [ ] Is this user story small enough to be completed in a Sprint? Should it be split?
8. [ ] Are the dependencies known/ understood? (technical, business, regulatory/policy)
9. [ ] Has the story been estimated?

**Definition of Done**

1.    [ ] Passes developer unit tests
2.    [ ] Passes peer code review
3.    [ ] If there's a user interface, passes UX assurance
4.    [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
5.	[ ] Confirm Test cases built and succeeding 
6.	[ ] No regression test failures
7.	[ ] Test coverage acceptable by Product Owner
8.	[ ] Ticket ready to be merged to master or story branch
9.	[ ] Developer to list Config changes/ Update documents and designs
10.	[ ] Can be demoed in Sprint Review 
11.	[ ] Tagged as part of a Release
12.   [ ] Feature flagged if required
13.  [ ] Change Management activities done?

**Comment by lmullane** [comment]
All scenarios passed QA.
